pages/linux/resize2fs.md

```diff
@@ -0,0 +1,17 @@
+# resize2fs
+
+> Resize an ext2, ext3 or ext4 file system.
+> Does not resize the underlying partition.
+> File system must be unmounted.
```

Comment:
Perhaps this will read better?
```suggestion
> Resize an ext2, ext3 or ext4 filesystem.
> Does not resize the underlying partition, and the filesystem must be unmounted.
```

This way we can avoid 3 lines of description.